dynamic-property-polling/src/main/java/ru/fix/dynamic/property/polling/DynamicPropertyPoller.java

```diff
@@ -0,0 +1,62 @@
+package ru.fix.dynamic.property.polling;
+
+/**
+ *
+ * @author Andrey Kiselev
+ */
+
+import java.util.WeakHashMap;
+import java.util.function.Supplier;
+import javax.annotation.PostConstruct;
+
+import ru.fix.dynamic.property.api.DynamicProperty;
+import ru.fix.stdlib.concurrency.threads.ReschedulableScheduler;
+import ru.fix.stdlib.concurrency.threads.Schedule;
+
+public class DynamicPropertyPoller {
+    private WeakHashMap<PolledProperty, Supplier> properties = new WeakHashMap<>();
+    private ReschedulableScheduler scheduler;
+    private DynamicProperty<Integer> rate;
+
+    public DynamicPropertyPoller(ReschedulableScheduler scheduler,
+                                 DynamicProperty<Integer> rate) {
```

Comment:
It would be better to provide DynamicProperty<Schedule> api instead - this way user can choose between rate and delay by itself. Usually delay is better choice,but there are cases, when it would be better to use fixed rate.

Reply:
ok